src/desktop/apps/auth2/components/MobileAuthStatic.tsx

```diff
@@ -17,26 +16,41 @@ interface Props {
 
 export class MobileAuthStatic extends React.Component<Props> {
   render() {
+    const submitUrls = {
+      login: '/log_in',
+      forgot: '/forgot_password',
+      signup: '/sign_up',
+      facebook: '/users/auth/facebook',
+      twitter: '/users/auth/twitter',
+    }
+
+    // TODO: pull analytics data
+    const authQueryData = {}
```

Comment:
Is this TODO blocking for launch?

Reply:
No, not for desktop. it will be a blocking todo for launching Mobile though